src/app/content/components/constants.ts

```diff
@@ -9,10 +9,24 @@ export const sidebarDesktopWidth = 33.5;
 export const sidebarMobileWidth = 28.8;
 export const sidebarTransitionTime = 300;
 
+export const searchResultsBarDesktopWidth = 43.5;
+export const searchResultsBarMobileWidth = 100;
+
 export const toolbarIconColor = theme.color.primary.gray;
 export const toolbarDesktopHeight = 5;
 export const toolbarMobileHeight = 4;
 
+export const mobileSearchContainerMargin = 1;
+export const toolbarSearchInputDesktopHeight = 3.2;
+export const toolbarSearchInputMobileHeight = 3.2;
+export const toolbarSearchInputMobileWidth = 18.5;
+export const toolbarToggleSearchBarTextMobileWidth = 11.2;
+export const toolbarMobileSearchWrapperHeight = toolbarSearchInputMobileHeight
++ (mobileSearchContainerMargin * 2) + 0.1;
```

Comment:
where does this `0.1` come from?

Reply:
From the <hr/> border height that separates the toolbar from the mobile search input